makezip.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -o errexit -o nounset
+
+MY_MODULES=$(git submodule status | awk '{print $2}' | tr '\n' ' ')
```

Comment:
What if we add a non-python submodule? You could check for the presence of `setup.py` in the subdirectory.

Reply:
Done.